### PR TITLE
test: add CLI timeout

### PR DESCRIPTION
**Pull request by erezrokah**
This PR adds a timeout of 30 seconds for CLI invocations in our tests.

### CODE REVIEW

File: tests/utils/call-cli.js [hunk 1]
```diff
@@ -2,8 +2,15 @@ const execa = require('execa')
 
 const cliPath = require('./cli-path')
 
+const CLI_TIMEOUT = 3e5
+
 const callCli = async function (args, execOptions) {
-  const { stdout } = await execa(cliPath, args, { windowsHide: true, windowsVerbatimArguments: true, ...execOptions })
+  const { stdout } = await execa(cliPath, args, {
+    windowsHide: true,
```
Comment (ehmicky) [inline tests/utils/call-cli.js:9]: This is already [the default value](https://github.com/sindresorhus/execa#windowshide).
Note: `windowsHide` is a tricky option which has [pros and cons](https://github.com/sindresorhus/execa/issues/433). We probably want to keep the default value here and let Execa decides what the best value for it is.

File: tests/utils/call-cli.js [hunk 2]
```diff
@@ -2,8 +2,15 @@ const execa = require('execa')
...
+    windowsVerbatimArguments: true,
```
Comment (ehmicky) [inline tests/utils/call-cli.js:10]: Is this needed? [The default behavior](https://github.com/sindresorhus/execa#windowsverbatimarguments) might be better at handling Windows.

**Review by ehmicky — APPROVED** [review]
Looks good.
I added two comments which are about options already present in the current code, not changed by this PR.

File: tests/utils/call-cli.js (hunk 1 above)
Comment (erezrokah) [reply tests/utils/call-cli.js]: Removed in https://github.com/netlify/cli/pull/1727/commits/130ee481e20c00e3d4c45f9fd3ceea59414b28e7

File: tests/utils/call-cli.js (hunk 2 above)
Comment (erezrokah) [reply tests/utils/call-cli.js]: Don't think it is. Removed in https://github.com/netlify/cli/pull/1727/commits/130ee481e20c00e3d4c45f9fd3ceea59414b28e7

**Pull request merged**